satellite/contact/benchmark_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package contact_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/satellite"
+	"storj.io/storj/satellite/overlay"
+	"storj.io/storj/satellite/satellitedb/satellitedbtest"
+)
+
+func BenchmarkUpdateCheckIn(b *testing.B) {
```

Comment:
Usually it's better to make an end-to-end test first then run the benchmark with the old code and then with the new code. Then you can compare the results with benchstat or benchcmp.

Reply:
oh neat, good idea. thanks!